examples/cctl/hello-python/readme.md

```diff
@@ -0,0 +1,95 @@
+## Example python Controller
+This example is taken from [metacontroller](https://metacontroller.app/guide/create/) doc. There were some issue in this example if you are using `CompositeController` and status subresource is not enabled for parent resource then your controller will go into an infinite loop. For `CompositeController` it updates `status.observedGeneration` with `metadata.generation` for parent resource.
+
+This will be done in 2 ways -
+
+1. If status subresource is enabled for parent then it will patch the status with cr's status end point and  `metadata.generation` of parent will not change.
+2. If status subresource is not enabled for parent then it will update the full object. For that there will be a change in `metadata.generation` of parent. Which will invoke a reconciliation and that reconciliation will invoke `metadata.generation` update in parent. So it will be a contineous loop.
+
+
+What changes you need to do? 
+
+You need to add below in your crd
+```yaml
+  subresources:
+    status: {}
+```
+### Use
+```yaml
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: helloworlds.example.com
+spec:
+  group: example.com
+  version: v1
+  names:
+    kind: HelloWorld
+    plural: helloworlds
+    singular: helloworld
+  scope: Namespaced
+  subresources:
+    status: {}
+```
+### Do not use
+```yaml
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: helloworlds.example.com
+spec:
+  group: example.com
+  version: v1
+  names:
+    kind: HelloWorld
+    plural: helloworlds
+    singular: helloworld
+  scope: Namespaced
+  subresources:
```

Comment:
What's the difference between these two yamls

Reply:
Forgot to remove. Updated thanks